zstacklib/zstacklib/test/test_zstack_daemon.py

```
@@ -0,0 +1,182 @@
+    sys.path.insert(0, _zstacklib_root)
+
+# ZStackDaemon only depends on `logger` from the module level of daemon.py.
+# Patch it before import so there is no real log file I/O.
+#
+# conftest.py (in the zstacklib/ parent dir) stubs out zstacklib.utils.daemon
+# as a plain MagicMock so GPU tests don't pull in the real daemon.  We must
+# pop that stub *before* importing the real module here.
+import types as _types
+
+_mock_log = _types.ModuleType('zstacklib.utils.log')
+_mock_log.get_logger = lambda name: mock.MagicMock()
```

Ruff (0.15.2)

[warning] Unused lambda argument: `name`

(ARG005)